src/storage/admin/AdminTaskManager.h

```diff
@@ -45,6 +53,10 @@ class AdminTaskManager {
   nebula::cpp2::ErrorCode cancelJob(JobID jobId);
   nebula::cpp2::ErrorCode cancelTask(JobID jobId, TaskID taskId = -1);
 
+  void cancelSpaceJobs(GraphSpaceID spaceId);
```

Comment:
because the parameter hit that we are handle jobs for specified space, then we can name these functions for short? sth like: cancelJobs(spaceId), runningJobCnt(spaceId), waitCancelJobs(spaceId).